app/components/Alerts/Front-End/Alert.tsx

```
@@ -0,0 +1,33 @@
+  const handleClose = () => {
+    if (dismissible) {
+      setVisible(false);
+    }
+  };
+
+  return (
+    visible && (
+      <div className={`alert alert-${type}`}>
+        {title && <h3>{title}</h3>}
+        <p>{message}</p>
+        {dismissible && <button onClick={handleClose}>Dismiss</button>}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)